plugins/openshift-developer/scripts/validate_precommit_config.py

```
@@ -0,0 +1,61 @@
+        "check-yaml",
+        "trailing-whitespace",
+    ],
+    "https://github.com/leaktk/gitleaks": None,  # all hooks allowed
+}
+
+
+def main():
+    try:
+        with open(".pre-commit-config.yaml") as f:
+            cfg = yaml.safe_load(f)
+    except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)